netlify.toml

```diff
@@ -3,11 +3,6 @@
   to = "/installing-spark/angular"
   status = 301
 
-[[redirects]]
-  from = "/gettingstarted/design-principles"
-  to = "/principles/design-principles"
```

Comment:
why get rid of the redirect? what if someone still has this url saved

Reply:
@RVMendoza  the page doesn't exist anymore. We deleted it. I think we could probably be good to remove all these redirects since its been a really long time since those links were active.